sriov_antrea_cni_install.sh

```diff
@@ -98,8 +98,8 @@ EOF
     git clone ${ANTREA_CNI_REPO} $WORKSPACE/antrea
     pushd $WORKSPACE/antrea
     if test ${ANTREA_CNI_PR}; then
-        git fetch --tags --progress ${ANTREA_CNI_REPO} +refs/pull/*:refs/remotes/origin/pr/*
-        git pull origin/pr/${ANTREA_CNI_PR}/head
+        git fetch --tags --progress ${ANTREA_CNI_REPO} +refs/pull/${ANTREA_CNI_PR}/*:refs/remotes/origin/pr/${ANTREA_CNI_PR}/*
```

Comment:
shouldnt this : `refs/remotes/origin/pr/${ANTREA_CNI_PR}/*`
be : `refs/remotes/origin/pull/${ANTREA_CNI_PR}/*`

looking at; https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/checking-out-pull-requests-locally

Reply:
actually this name (`refs/remotes/origin/pr/${ANTREA_CNI_PR}/*`) is the local counterpart of the PR (`BRANCHNAME` in the link you sent), so it can actually be any name. regarding the next line in which we do pull, we actually pull from the remote branch, so functionally, all would be working, though it would be better of course if we did it as you said to have the local branch named as the remote PR. 

also, i did some testing, concluding that this line can be enhanced if we don't want to checkout the PR and we only need to pull it. we can only do `git fetch --tags --progress ${ANTREA_CNI_REPO} pull/${ANTREA_CNI_PR}/head`, then pull it in the next line without creating a local branch for it. The reason of why were we doing it the way it is now is that, this is a way (a hack if you may) to pull all PRs from remote in a single line. then either pull them or check them out (you can not checkout the branch if you just fetch it without creating a local branch for it).

Reply:
OK so you just need to fetch/pull just the PR you are working on IMO.
can you make the remote and local refs the same e.g both to use .../pull/... ?

also lets update commit message and im LGTM